### ISSUE TITLE
Search appears not to work right now

### ISSUE THREAD

**Issue by tedivm** [opentofu/registry-ui#227]
### URL of the page that's broken

https://search.opentofu.org/

### Screenshot

<img width="1820" alt="Screenshot 2024-09-23 at 1 56 58 PM" src="https://github.com/user-attachments/assets/ee5d74e4-af2d-4e81-b78f-61c31b3bdd00">

### OS, browser version, installed extensions

Firefox 129.0.1 (64-bit)

MacOS 14.6.1

### Additional information

When I try to search for something it clears the search field but nothing happens- there isn't even a network call.

**Comment by cam72cam** [comment]
That's odd, it's working fine for me (FF 129.0.2 (64-bit))

**Comment by tedivm** [comment]
I think my issue is that I type too fast. If you go to the page, type in a provider name, and hit enter before the autocomplete thing shows up then the search field gets emptied and no auto complete occurs (or actual search). You need to tweak the javascript so that you take over the "enter" command and don't clear the field.

**Comment by cam72cam** [comment]
Ah, I've hit that myself.  That's definitely a weird UI quick that needs to be fixed.

**Comment by tedivm** [comment]
No, the same bug is there.

> I think my issue is that I type too fast. If you go to the page, type in a provider name, and hit enter before the autocomplete thing shows up then the search field gets emptied and no auto complete occurs (or actual search). You need to tweak the javascript so that you take over the "enter" command and don't clear the field.

You should change the javascript to ignore when someone hits the enter key to resolve this bug.

**Comment by diofeher** [comment]
I tried to type fast in this one, but it worked as expected.

Strangely enough, accessing this link above (https://search.opentofu.org/provider/databricks/databricks/resources/global_init_script.md) showed me "An error occured".

So I went to click on every link of this page that ended with .md, and all of them are broken. 

What are we expected to show in these markdown links?

example:
<img width="959" alt="Screenshot 2025-01-03 at 09 21 24" src="https://github.com/user-attachments/assets/7dbd59e8-5c6a-4d6f-b38d-956196cc2f60" />

Broken links:

https://search.opentofu.org/provider/databricks/databricks/latest/docs/data-sources/spark_version.md
https://search.opentofu.org/provider/databricks/databricks/data-sources/dbfs_file.md

Basically all md links are broken

**Comment by diofeher** [comment]
@tedivm are you still able to see this bug? I think we've fixed it along with a few different changes on the UI. I tested right now and wasn't able to reproduce.

**Comment by tedivm** [comment]
It's fixed!
